### ISSUE TITLE
Add Machine & Deep Learning Notebooks with Datasets

### ISSUE THREAD

**Issue by kunalagra** [Mind-Benders/ML_DL_Projects#1]
Hello Contributors,

For this repository, if you have your work where you have performed Machine/Deep Learning on a datasets, we would love to have them on our Project Repo.

TO Contribute your project:
- Should be in Python or R
- The Code should be well commented from a perspective of beginner
- If you are using online dataset, please link the relevant websites
- Visualization is not necessary but would be good to have
- No Direct COPY-PASTING of others work.
- Your Notebook should reside inside a folder at ROOT of repo. (Create a new folder with Appropriate Name and then inside it add your Jupyter/R Notebook)
- At End of notebook, please consider adding your details (name, github/social links)
- Follow [Code Of Conduct ](https://github.com/Mind-Benders/.github/blob/main/profile/code_of_conduct.md) & [CONTRIBUTING](https://github.com/Mind-Benders/.github/blob/main/profile/CONTRIBUTING.md) guidelines

Steps:
- Open a Issue with appropriate Title or comment your Title
- Open a relevant PR
- Make changes if required and Enjoy!

**Comment by Aryaman047** [comment]
Can you plz assign this issue to me

**Comment by kunalagra** [comment]
> Can you plz assign this issue to me

Hey @AryamanAdivyaSingh47,
Could you please open appropriate Issue which will detail what you will be adding to the repo.
It should include your name, dataset, idea for implementations too.

**Comment by Aryaman047** [comment]
> > Can you plz assign this issue to me
> 
> Hey @AryamanAdivyaSingh47, Could you please open appropriate Issue which will detail what you will be adding to the repo. It should include your name, dataset, idea for implementations too.

Ok, so I will be training a deep Learning model using LSTM to create a model that translates german to English.

**Comment by kunalagra** [comment]
> > > Can you plz assign this issue to me
> > 
> > 
> > Hey @AryamanAdivyaSingh47, Could you please open appropriate Issue which will detail what you will be adding to the repo. It should include your name, dataset, idea for implementations too.
> 
> Ok, so I will be training a deep Learning model using LSTM to create a model that translates german to English.

That Sounds Great. I have opened #3. Please ask for assignment there. You can followup with your progress/questions over there.

**Comment by Adw8** [comment]
@kunalagra Hi! I have worked on a custom object detector project in the form of a colab notebook, Can I contribute that to the repository in the form of an issue?

**Comment by kunalagra** [comment]
> @kunalagra Hi! I have worked on a custom object detector project in the form of a colab notebook, Can I contribute that to the repository in the form of an issue?

Yes. @Adw8 You can. Just create a new issue with relevant titles and description.
